### ISSUE TITLE
Renew slice modifies the expiration time

### ISSUE THREAD

**Issue by innnkkki** [GENI-NSF/geni-tools#60]
When trying to renew a slice against a gcf SA, it will renew it to be a little bit less than the time specified, which makes it a little bit
harder to renew slivers later, since you need to specify a new time for your renewsliver command. 
```
nriga@pella:~/gcf$ src/omni.py renewslice -f orca_gcf orcaslice 20110820T19:00:00Z
INFO:omni:Loading config file /home/nriga/.gcf/omni_config
INFO:omni:Using control framework orca_gcf
INFO:omni:Slice orcaslice now expires at 2011-08-20 19:00:00+00:00 UTC
INFO:omni:Slice urn:publicid:IDN+geni:gpo:gcf+slice+orcaslice expires on 2011-08-20 18:59:59 UTC
INFO:omni: ------------------------------------------------------------
INFO:omni: Completed renewslice:

  Options as run:
		framework: orca_gcf
		native: True

  Args: renewslice orcaslice 20110820T19:00:00Z

  Result Summary: Slice orcaslice now expires at 2011-08-20 19:00:00+00:00 UTC
Slice urn:publicid:IDN+geni:gpo:gcf+slice+orcaslice expires on 2011-08-20 18:59:59 UTC 
INFO:omni: ============================================================
```

 _Imported from trac ticket [#60](http://trac.gpolab.bbn.com/gcf/ticket/60),  created by nriga on 08-17-2011 at 17:29, last modified: 08-25-2011 at 10:24_

**Comment by tcmitchell** [comment]
The issue is that the sample clearinghouse is dropping the microseconds after a time difference, causing the expiration to be off by a little under a second. Rather than passing a delta along, the clearinghouse should pass along the requested expiration time.

_Trac comment by tmitchel (github user: tcmitchell) on 08-25-2011 at 10:13_

**Comment by tcmitchell** [comment]
Fixed in [e1f4710a91c0ab814186b261a59fe16ac43d697c](https://github.com/GENI-NSF/geni-tools/commit/e1f4710a91c0ab814186b261a59fe16ac43d697c)


_Trac comment by tmitchel (github user: tcmitchell) on 08-25-2011 at 10:24_
